lib/ui/pages/main/home/home.dart

```diff
@@ -209,3 +75,102 @@ class HomePage extends StatelessWidget {
     );
   }
 }
+
+class MyTile extends StatelessWidget {
```

Comment:
少しfileが必要以上に肥大化しいてるように思えるので、個別のWidgetは `/Widgets/`を作ってその中に入れても良さそうですね 🙆‍♂️ 
このままなら
```suggestion
class _MyTile extends StatelessWidget {
```
とできそうでしょうか？